charts/datahub/values.yaml

```diff
@@ -754,7 +754,7 @@ global:
       #   value: password
 
   datahub:
-    version: v0.15.0.1
+    version: v1.0.0
```

Comment:
@pedro93 is there a feature flag that OSS users can enable for the new UI?

Reply:
Not built into the charts. I'll add it